app-ios/Modules/Sources/Assets/swiftgen.yml

```diff
@@ -23,6 +23,15 @@ json:
       enumName: LottieAssets
       publicAccess: true
 
+fonts:
+  inputs: ../../../../../core/designsystem/src/commonMain/resources/font
+  outputs:
+    templateName: swift5
+    output: Fonts.swift
+    params:
+      enumName: FontAssets
+      publicAccess: true
```

Comment:
Can you introduce font with `Theme` module? 👀 
I think font is related to theme.

Reply:
@ry-itto 
Sorry if I have made a recognition error. :bow:
I am aware that this is the content of task #889 , is that correct?
I would like to define typography as a theme in task #889  after introducing it to the iOS project.

Reply:
@ry-itto 
Please let me confirm. :bow: 
Does this mean you want it defined in Theme/swiftgen.yml instead of Assets/swiftgen.yml?

Reply:
> Does this mean you want it defined in Theme/swiftgen.yml instead of Assets/swiftgen.yml?

Yes 🙏🏼

Reply:
@ry-itto 
Thank you very much for your patience. 🙇 
Fixed to have Font resource in Theme.